scripts/pr-infra-compose.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+# This script enables the deployment of a dedicated app instance using the
```

Comment:
It would be nice to add a bit of doc for this.
I'd go for an update of the website https://osrd.fr/en/docs/guides/deploy/docker-compose (the localhost access part of components there should probably be updated too, mentioning that it's only available when using `docker/docker-compose.host.yml` file).

Reply:
Hey, I'll update the doc indeed, you can still access (nearly) everything from loccalhost, it's just that the port you need to use has been moved (using a +1 operation, or +2 on two cases when the +1 was already taken).
The only thing that is not accessible is the front as it is served directly by the gateway and not as a service on it's own.

Reply:
However, I would say that it's bet to talk about the script here https://osrd.fr/en/docs/guides/contribute/code-review/ as it is intended for code review usage only.
That what I'll do for now but we can always debate on this and move it later if you think it would be best otherwhere.

Reply:
Good idea to at least mention it in code-review part, yes.
As long as things are documented only once, it's fine for me (the place and the links are easier to maintain/add).

As for the update I mentioned, about being able to request any component: it was a mistake of mine, sorry, I mixed things and it's already good. I edit my first message.

Reply:
I made a PR on that, I'll ping you in it so you can have a look.
https://github.com/OpenRailAssociation/osrd-website/pull/241